network/controller/bootstrap/helper.go

```diff
@@ -124,3 +124,20 @@ func GenerateNonce() (Nonce, error) {
 	}
 	return buffer[:], nil
 }
+
+// FindDiscoveryInActiveNodeList returns only discovery nodes from active node list
+func FindDiscoveryInActiveNodeList(activeNodes []core.Node, cert core.Certificate) []core.Node {
```

Comment:
the function returns a list of nodes, so `FindDiscoveriesInActiveNodeList`

Reply:
fixed